cmd/common-main.go

```diff
@@ -778,6 +778,8 @@ func handleCommonEnvVars() {
 	}
 
 	globalDisableFreezeOnBoot = env.Get("_MINIO_DISABLE_API_FREEZE_ON_BOOT", "") == "true" || serverDebugLog
+
+	globalEnableSyncEvents = env.Get("MINIO_ENABLE_SYNC_EVENTS", "") == "true"
```

Comment:
We should make this an option in the API subsystem?

MINIO_API_SYNC_EVENTS

Reply:
sure, will do this change